package.json

```diff
@@ -22,7 +22,9 @@
     "test": "ava '*.test.js' '**/*.test.js' '!integration_test/*'",
     "test:integration": "npm run db:init && ava --concurrency=1 integration_test/*.test.js",
     "test:cover": "nyc --reporter=text-lcov ava --concurrency=1 '*.test.js' '**/*.test.js' > coverage.lcov && codecov",
+    "test:inta": "npm run db:init && ava --concurrency=1 integration_test/auth.integration.test.js",
```

Comment:
IMO, a longer name like `test:auth-integration` would be better.

Reply:
I won't remove it, but rename it to `test:integration:auth`. It's useful to run this part only.